### PR TITLE
[Error-Catching]: Added feature of handling error in javascript.

### PR DESCRIPTION
**Pull request by fayzzzm**

### CODE REVIEW

**Comment by fayzzzm** [comment]
As soon as possible, I will write tests for the functions that I've provided. For now, I added a feature of handling a javascript error.

File: src/client/src/components/FileButton/styles.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
     padding: 8px 14px;
     color: #fff;
     height: 30px;
-    font-family: monospace;
+    font-weight: bold;
```
Comment (VoIdJMp) [inline src/client/src/components/FileButton/styles.scss:5]: It was also possible to add cursor: pointer when hovering over the button

File: src/server/files/internal.js [hunk 2]
```diff
@@ -1,4 +1,29 @@
...
+console.warn = (...content) => {
+    logInterceptor('log--warn', content);
+};
+
+window.onerror = function (msg) {
+    console.error(msg);
+
+    return false;
+};
+
```
Comment (VoIdJMp) [inline src/server/files/internal.js:20]: There is also console.info which also needs to be added

Comment (fayzzzm) [reply src/server/files/internal.js:20]: Yeah, sure. For now, for the 1 section, we just need to support `log | error | warning`. The others could be in the future version of the product.

File: src/client/src/components/FileButton/styles.scss (hunk 1 above)
Comment (fayzzzm) [reply src/client/src/components/FileButton/styles.scss:5]: Agree. For now, we created the MVP of the product, so those changes could be done in the next versions of the product.

File: src/server/files/internal.js (hunk 2 above)
Comment (VoIdJMp) [reply src/server/files/internal.js:20]: Ok

**Pull request merged**